src/main/java/eu/europeana/metis/sandbox/service/workflow/HarvestService.java

```diff
@@ -0,0 +1,43 @@
+package eu.europeana.metis.sandbox.service.workflow;
+
+import eu.europeana.metis.sandbox.common.exception.ServiceException;
+import java.io.ByteArrayInputStream;
+import java.util.List;
+import org.springframework.web.multipart.MultipartFile;
+
+
+public interface HarvestService {
+
+  /**
+   * Harvest the given file {@link MultipartFile} to a list of byte[], one string per file in the
```

Comment:
Every javadoc in here have the same description (with the exception of the variables descriptions), could you change them to be more specific?

Reply:
More specific documentation of the harvester library is available in the metis-framework harvester class javadoc. If you believe it is beneficial to add information from there, here, sure. I believe the current javadoc is enough

Reply:
The changes done here are what I wanted, so it's fine as it is